modules/apps/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/js/PermissionNavigationBar.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import ClayLink from '@clayui/link';
+import ClayNavigationBar from '@clayui/navigation-bar';
+import React from 'react';
+export default function PermissionNavigationBar({
+	navigationItems: _navigationItems,
+	portletNamespace: _portletNamespace,
+}) {
+	const alertMessage = document.getElementById(
+		_portletNamespace + 'alertMessage'
+	);
+	const saveButton = document.getElementById(
+		_portletNamespace + 'saveButton'
+	);
+	function onChangeTab(event, href) {
+		if (!alertMessage.classList.contains('hide')) {
+			openPopUp(href);
+		}
+		else {
+			window.location.href = href;
+		}
```

Comment:
The popup will not be shown when editing the permissions of a folder with the propagation setting set to false. Please double-check. Thanks.